app/controllers/ITSAStatusController.scala

```diff
@@ -43,6 +43,7 @@ class ITSAStatusController @Inject()(authentication: AuthenticationPredicate,
       case Left(error: ITSAStatusResponseError) =>
         logger.error(s"[ITSAStatusController][getITSAStatus] - Error Response: $error")
         Status(error.status)(Json.toJson(error))
+      case Left(_) => InternalServerError(s"[ITSAStatusController][getITSAStatus]")
```

Comment:
Please add log entry as well, as this would be hard to investigate potential issue is needed